hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnectionErrorHandler.java

```diff
@@ -55,11 +55,12 @@ synchronized void onError(Throwable cause) {
         }
     }
 
-    synchronized void reset() {
+    synchronized TcpServerConnectionErrorHandler reset() {
```

Comment:
I prefer the old style `void reset()` as it is more consistent with other `reset` usages around the codebase, but that's just me

Reply:
I'll revert. I had some plans with that reset, which then didn't materialize, but still, I left it there.

Reply:
Usually I'm all for consistency, but in this case I would say that better not violate the single responsibility principle. Cleaner this way.

Reply:
I prefer it returning `void` too. How do we violate the SRP here? And I don't see the return value being actually used.

Reply:
![image](https://user-images.githubusercontent.com/6204813/118644409-6fbdc180-b7e6-11eb-9c0f-437dae2ea7a9.png)

This getter should get the error handler and that's it, not have a reset as a side effect inside.

Reply:
The return value is just for the ability to chain calls together.